core/templates/dev/head/pages/exploration-editor-page/services/change-list.service.ts

```diff
@@ -47,8 +47,8 @@ oppia.factory('ChangeListService', [
     var CMD_DELETE_STATE = 'delete_state';
     var CMD_EDIT_STATE_PROPERTY = 'edit_state_property';
     var CMD_EDIT_EXPLORATION_PROPERTY = 'edit_exploration_property';
-    var CMD_IMAGE_ASSETS = 'image_assets'
-    var IMAGE_ASSETS_ACTION = 'add_image'
+    var CMD_IMAGE_ASSETS = 'image_assets';
```

Comment:
I don't understand. I thought we were planning to have cmd = CMD_EDIT_EXPLORATION_PROPERTY, property = image_assets and action = add_image or delete_image. Why do we have a cmd image_assets?

/cc @DubeySandeep I recollect discussing the above mentioned details, could you confirm?

Reply:
yeah, your confusion is right. Earlier we were implementing like the way you have told.

But after the merging of Ankita  PR, there is the change in Plan.
Ankita PR has introduced some generalized way for initializing change command.
You can see ```ALLOWED_COMMANDS ``` in ```exp_domain.py ```, where we were expecting that 
``` cmd = CMD_EDIT_EXPLORATION_PROPERTY ``` should always contains new value.

But since if we use cmd = CMD_EDIT_EXPLORATION_PROPERTY and action = add_image then there is no need of new value. We only need image info in change dict. so to overcome this I create a new command for image assets.

But after some more thinking I want to ask one thing, can we create change dict to be this considering old plan
```
cmd: CMD_EDIT_EXPLORATION_PROPERTY,
state_name: state_name
property_name: image_assets,
new_value: image_info
action: add_image
```

Reply:
ping @DubeySandeep